client/my-sites/stats/stats-list/stats-list-item.jsx

```diff
@@ -216,8 +230,24 @@ const StatsListItem = React.createClass( {
 			}
 
 			if ( data.link ) {
+				let href = data.link;
+				let onClickHandler = this.preventDefaultOnClick;
+				if ( this.isFollowersModule ) {
+					const siteId = this.getSiteIdForFollow();
+					if ( siteId ) {
+						href = '';
```

Comment:
why do we erase the href?

Reply:
i don't think we need to since i've added the `preventDefault()`

Reply:
over in reader, we only prevent when it's a default (left) mouse click with no modifiers held. That way, a user can option-click or right click on the link and open the original URL in a new tab or window.

Reply:
ah, ok ill update this